### PR TITLE
Use @digitalbazaar/http-digest-header

### PR DESCRIPTION
**Pull request by JSAssassin**
Removed support for node 10 since it doesn't support TextEncoder.
addresses #14

### CODE REVIEW

**Comment by aljones15** [comment]
@JSAssassin can you try using this branch with a corresponding `verify` branch in a project and let me know if the http-signatures continue to work? If you don't have time I'll try  it myself.

**Comment by JSAssassin** [comment]
@aljones15 I can do that, but I'm not sure which `verify` branch you are talking about and in which project?

File: main.js [hunk 1]
```diff
@@ -4,11 +4,10 @@
 'use strict';
 
 import base64url from 'base64url-universal';
-import crypto from './crypto.js';
 import pako from 'pako';
 import {TextEncoder, URL, base64Encode} from './util.js';
 import {createAuthzHeader, createSignatureString} from 'http-signature-header';
-
+import {createHeaderValue} from '@digitalbazaar/http-digest-header';
```
Comment (dlongley) [inline main.js:10]: Move `import` up a line and keep whitespace line after it.

File: main.js [hunk 2]
```diff
@@ -75,16 +74,8 @@ export async function signCapabilityInvocation({
...
-    // format as multihash digest
-    // sha2-256: 0x12, length: 32 (0x20), digest value
-    const mh = new Uint8Array(34);
-    mh[0] = 0x12;
-    mh[1] = 0x20;
-    mh.set(digest, 2);
-    // encode multihash using multibase, base64url: `u`
-    signed.digest = `mh=u${base64url.encode(mh)}`;
+    signed.digest = await createHeaderValue(
+      {data, useMultihash: true});
```
Comment (dlongley) [inline main.js:78]:
```suggestion
    signed.digest = await createHeaderValue({data: json, useMultihash: true});
```

File: main.js [hunk 3]
```diff
@@ -75,16 +74,8 @@ export async function signCapabilityInvocation({
   if(json && !('digest' in signed)) {
     // compute digest for json
     const data = new TextEncoder().encode(JSON.stringify(json));
```
Comment (dlongley) [inline main.js:76]: The `createHeaderValue` library can handle this for us now, so just pass `json` to it below.

```suggestion
```

**Review by dlongley — CHANGES_REQUESTED** [review]

**Review by dlongley — APPROVED** [review]
Thanks!

**Review by aljones15 — APPROVED** [review]
LGTM feel free to merge.

**Review by dmitrizagidulin — APPROVED** [review]
👍

**Pull request merged**